### ISSUE TITLE
Node-red projects enabled in settings.js, but disabled in Node-red 

### ISSUE THREAD

**Issue by iZac85** [hassio-addons/addon-node-red#995]
# Problem/Motivation

I'm using Projects (https://nodered.org/docs/user-guide/projects/#introducing-projects) in Node-red for backup and version control (git). However since updating to v.9.0.0 Node-red projects are not longer enabled in the Node-red menu and old flows located in /config/node-red/ are loaded instead. 

## Expected behavior

If projects are enabled in settings.js I expect Node-red to use projects. 

## Actual behavior

I have projects enabled in my settings.js:
```
  // Customising the editor
  editorTheme: {
    projects: {
      // To enable the Projects feature, set this value to true
      enabled: true
    }
  }
```

However the log shows that Node-red does not detect that projects are enabled and I cannot use projects in Node-red:

```
> node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"

22 Apr 09:47:09 - [info] 

Welcome to Node-RED
===================

22 Apr 09:47:09 - [info] Node-RED version: v1.3.3
22 Apr 09:47:09 - [info] Node.js  version: v14.16.1
22 Apr 09:47:09 - [info] Linux 5.10.17-v7l+ arm LE
22 Apr 09:47:10 - [info] Loading palette nodes
22 Apr 09:47:15 - [info] Dashboard version 2.28.2 started at /endpoint/ui
/bin/sh: /opt/node_modules/node-red-node-pi-gpio/testgpio.py: not found
22 Apr 09:47:16 - [warn] rpi-gpio : Raspberry Pi specific node set inactive
22 Apr 09:47:16 - [info] Settings file  : /etc/node-red/config.js
22 Apr 09:47:16 - [info] Context store  : 'default' [module=memory]
22 Apr 09:47:16 - [info] User directory : /config/node-red/
22 Apr 09:47:16 - [warn] Projects disabled : set editorTheme.projects.enabled=true to enable
22 Apr 09:47:16 - [info] Flows file     : /config/node-red/flows.json
```

## Steps to reproduce

- Enable projects in /config/node-red/settings.js (see https://nodered.org/docs/user-guide/projects/#enabling-projects)
- Start Node-red add-on
- Check in the log if projects are not enabled (**_[warn] Projects disabled : set editorTheme.projects.enabled=true to enable_**)

## Proposed changes

Something changed between version v0.8.1 and v.9.0.0 that cause this issue. 
If the setting could be added to the configuration for the add-on that would be nice.

**Comment by PhilippCh** [comment]
This has already been fixed in #992, but no patch has been released so far.

**Comment by frenck** [comment]
Fixed in v9.0.1
